src/main/java/io/litmuschaos/LitmusClient.java

```diff
@@ -18,181 +19,176 @@
 import java.util.List;
 import java.util.Map;
 
+import static io.litmuschaos.constants.ApiEndpoints.*;
+import static io.litmuschaos.constants.RequestParams.*;
+
 public class LitmusClient implements AutoCloseable {
 
-    private String token;
+    private LitmusAuthToken token;
     private final LitmusHttpClient httpClient;
     private final LitmusGraphQLClient graphQLClient;
 
-    public LitmusClient(String host, String token) {
+    public LitmusClient(String host, LitmusAuthToken token) {
```

Comment:
```suggestion
    public LitmusClient(String host, String token) {
```

Can you get token as string and make like below?

this.token = new ListAuthToken(token);